app/models/profile.rb

```diff
@@ -3,10 +3,8 @@
 class Profile < ApplicationRecord
   belongs_to :user
 
-  validates :name, presence: true
   validates :name, length: { maximum: 100 }
```

Comment:
```suggestion
  validates :name, length: { maximum: 50 }
```

Reply:
@nguyenphuhongloan Why is the name limited to only 50 characters?

Reply:
50 is enough

Reply:
@nguyenphuhongloan oke boss